tests/test-meta-revisions.php

```diff
@@ -14,6 +14,10 @@ class MetaRevisionTests extends WP_UnitTestCase {
 
 	/**
 	 * Callback function to add the revisioned keys.
+	 *
+	 * @param Array $keys The passed array of keys to revision.
```

Comment:
```suggestion
	 * @param array $keys The passed array of keys to revision.
```

Reply:
👍 I can't commit this in github for some reason :)

Reply:
Weird. Might be because you didn't initiate this PR. I'll just commit those changes in a separate commit in one go now.

Reply:
thanks